### ISSUE TITLE
Upgrade Gnome runtime to version 40

### ISSUE THREAD

**Issue by matteo-gallo-bb** [flathub/org.gnome.Fractal#18]
Please upgrade Gnome runtime to version 40 and org.freedesktop.Platform.ffmpeg-full to version 21.08.

**Comment by hfiguiere** [comment]
not ffmpeg-full, GNOME 40 still use 20.08

**Comment by hfiguiere** [comment]
As for the runtime, see #17

**Comment by matteo-gallo-bb** [comment]
I was that PR but we can upgrade to Gnome 40 anyways without removing libhandy dependency. This is useful because all other Flatpak apps are migrating to Gnome 40, so we can free up space removing "old" runtimes, in this case Gnome 3.38.

**Comment by tim77** [comment]
> so we can free up space removing "old" runtimes

I like this idea as well to get rid of old runtimes and update to new one if there is no problems. Also 3.38 is EOL already so we MUST update on newer version. I've rebased old PR and bumped runtime to v41, some modules and don't touched libhandy at this point. Testing right now this build and LGTM so far.

Let's merge #17 if there is no issues? I've got [this](https://gitlab.gnome.org/GNOME/fractal/-/issues/723#note_1211364) but this seems like upstream issue which hopefully will be fixed soon.
